sql/sql_parse.cc

```diff
@@ -7416,17 +7398,17 @@ static bool wsrep_should_retry_in_autocommit(enum_sql_command &sql_command) {
   }
 }
 
-static bool wsrep_mysql_parse(THD *thd, const char *rawbuf, uint length,
+static bool wsrep_dispatch_sql_command(THD *thd, const char *rawbuf, uint length,
                               Parser_state *parser_state,
                               bool update_userstat) {
-  DBUG_ENTER("wsrep_mysql_parse");
+  DBUG_ENTER("wsrep_dispatch_sql_command");
```

Comment:
Minor comment: if we are changing these DBUG_ENTERs, we could use DBUG_TRACE instead.